hypervisor/src/mshv/mod.rs

```diff
@@ -1495,17 +1504,16 @@ impl MshvVcpu {
     /// Clear SW_EXIT_INFO1 register for SEV-SNP guests.
     ///
     #[cfg(feature = "sev_snp")]
-    fn clear_swexit_info1(
-        &self,
-        ghcb_gpa: u64,
-    ) -> std::result::Result<cpu::VmExit, cpu::HypervisorCpuError> {
+    fn clear_swexit_info1(&self) -> std::result::Result<cpu::VmExit, cpu::HypervisorCpuError> {
         // Clear the SW_EXIT_INFO1 register to indicate no error
-        self.gpa_write(ghcb_gpa + GHCB_SW_EXITINFO1_OFFSET, &[0; 4])?;
+        let ghcb_page: *mut svm_ghcb_base = self.ghcb_addr.unwrap() as *mut svm_ghcb_base;
+        set_svm_field_u64_ptr!(ghcb_page, exit_info1, 0);
 
         Ok(cpu::VmExit::Ignore)
     }
 
     #[cfg(feature = "sev_snp")]
+    #[allow(dead_code)]
```

Comment:
Why is there a need for this now?

Reply:
Sorry. This API is not used any more. Should I just remove it?

Reply:
Yes. It should be removed.